### ISSUE TITLE
Updated APK?

### ISSUE THREAD

**Issue by IzzySoft** [simra-project/simra-android#23]
Are you still updating the APK on your website? The URL still returns `versionCode` 81, while your repo already says 88. Thus your app in my repo (for those outside PlayStore) is quite outdated meanwhile.

**Comment by IzzySoft** [comment]
While on it: for 81, my library scanner reports the presence of Sentry.io – which is why the `Tracking` Anti-Feature was applied automatically. How is Sentry configured with your app: is it opt-in (so the user gets to choose **before** any data is sent)? Do you self-host your Sentry instance? Maybe the Anti-Feature could be lifted.

**Comment by selphiron** [comment]
yeah, the link to the APK changed. It is now https://tubcloud.tu-berlin.de/s/eK46JHtY7F2exq7/download
I updated the Link on our "mitmachen" page.
Regarding logging: it is opt-in and we self-host it. I think I can remove Sentry from the gradle

**Comment by IzzySoft** [comment]
Ouch, that new URL does not work with my updater:

    de.tuberlin.mcc.simra.app: Server does not give Last-Modified in header, skipping.

Any chance to get that fixed? Manual updates will be much delayed as I only find time for them occasionally. Ideally you'd [tag your releases](https://github.com/simra-project/simra-android/releases) and have the APKs attached there…

And glad to read about your Sentry setup – opt-in is fine, especially when self-hosted! I've adjusted metadata at my end accordingly to no longer point out the "potential tracking" :wink:

**Comment by IzzySoft** [comment]
And it looks like the APK link has changed again: on the above I still get v88, while Play has v90 since November. And the link given on the website is broken (server permanently resets the connection). Why not simply attaching the APKs to their corresponding [release](https://github.com/simra-project/simra-android/releases) here, @selphiron ?

**Comment by IzzySoft** [comment]
@selphiron any chance?

**Comment by selphiron** [comment]
Sorry for the very late answer @IzzySoft 
Here is the link to v92:
https://tubcloud.tu-berlin.de/s/FxfJipgBfnqTbr4/download/92.apk
I will try to update the link on the project website ASAP

**Comment by IzzySoft** [comment]
Thanks! Just pulled the APK, will show up in my repo with the next sync.
